src/procedure/examples/output/uniqueDevice/output.ts

```diff
@@ -1,10 +1,20 @@
 import { outputPort } from '../outputPort'
 import type { SerialPort } from 'serialport'
+import { delay } from '../../../utils/delay'
 
 export const attachOutput = (port: SerialPort, pin: number) => {
   const output = outputPort(port)(pin)
 
   return {
+    blink: async (duration: number) => {
+      // eslint-disable-next-line no-constant-condition
+      while (true) {
+        await output.on()
+        await delay(duration)
+        await output.off()
+        await delay(duration)
+      }
+    },
     on: async () => {
```

Comment:
Do not use output.ts with the exception of LEDs only. We do not want to be able to call blink when developers are dealing with output devices.